atlas/templates/cmd/server/main.go.gotmpl

```diff
@@ -58,6 +58,7 @@ func NewLogger() *logrus.Logger {
 
 	// Set the log level on the default logger based on command line flag
 	logLevels := map[string]logrus.Level{
+		"trace":   logrus.TraceLevel,
```

Comment:
A better solution would be to replace this whole section with https://pkg.go.dev/github.com/sirupsen/logrus#ParseLevel

Reply:
done.